jobs/core/templates/bin/vault

```diff
@@ -20,13 +20,19 @@ case $1 in
       rm -f $PIDFILE
     fi
 
-    if chpst -u vcap:vcap /var/vcap/packages/vault/bin/canimlock; then
-      sed -i -e 's/^disable_mlock/#disable_mlock/' \
-        $JOB_DIR/config/vault.conf
-    else
-      sed -i -e 's/^#disable_mlock/disable_mlock/' \
-        $JOB_DIR/config/vault.conf
-    fi
+    #if chpst -u vcap:vcap /var/vcap/packages/vault/bin/canimlock; then
```

Comment:
We should remove the commented-out blocks of code and allow versioning to be our historical reference.

Reply:
I removed the commented lines.